api/app/signals/apps/api/views/signal.py

```diff
@@ -258,11 +261,21 @@ def history_log(self, *args, **kwargs):
         serializer = HistoryLogHalSerializer(history_log_qs, many=True)
         return Response(serializer.data)
 
-    @action(detail=False, url_path=r'geography/?$')
+    @action(detail=False, url_path=r'geography/?$', filterset_class=SignalFilterSet)
```

Comment:
I thought we were already using general filtering here

Reply:
It was not working properly, and this way it is more readable we are using the same filtering as the private signal endpoint